ui/src/App.tsx

```diff
@@ -80,12 +103,27 @@ const setup = async () => {
     issuers => {
       console.log('Got issuers', issuers);
       useAppStore.setState({
-        issuers,
+        issuers: new Map(issuers),
       });
     }
   );
 };
 
+const Banner = () => (
```

Comment:
This probably isn't necessary, it would free up some real estate if we just removed it